src/float/surface_transceiver/surface_transceiver.ino

```diff
@@ -159,8 +158,8 @@ bool receivePacket() {
       byteBuffer[PKT_IDX_PROFILE_HALF]);
     for (int i = PKT_HEADER_LEN; i < len;) {
       // Wonky pointer casting to convert four bytes into a long (time)
-      serialPrintf("%l,", *reinterpret_cast<uint32_t*>(byteBuffer + i));
-      i += sizeof(uint32_t);
+      serialPrintf("%l,", *reinterpret_cast<unsigned long*>(byteBuffer + i));
+      i += sizeof(unsigned long);
```

Comment:
Times are specifically returned as `unsigned long`s in Arduino

Reply:
?